sormas-api/src/main/java/de/symeda/sormas/api/systemevents/SystemEventType.java

```diff
@@ -1,6 +1,6 @@
 package de.symeda.sormas.api.systemevents;
 
 public enum SystemEventType {
-	FETCH_LAB_MESSAGES,
+	FETCH_EXTERNAL_MESSAGES,
```

Comment:
This then requires a database migration, doesn't it? Each stored SystemEvent has a type...

Reply:
true